wire-ios-data-model/Source/Model/User/TypeOfUser.swift

```diff
@@ -18,12 +18,19 @@
 
 @objc(ZMTypeOfUser)
 public enum TypeOfUser: Int16, CaseIterable, Sendable {
-    /// Regular users
```

Comment:
The cherry-pick PR https://github.com/wireapp/wire-ios/pull/3790 is/was not yet merged. Once it is merged these changes should be gone.